plugins/clusters.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: clusters
+spec:
+  version: "v0.1.1"
+  platforms:
+  - uri: https://github.com/u2takey/kubectl-clusters/archive/v0.1.1.tar.gz
+    sha256: 369dcf9c83f38b0068a5176b802285bfe4cfda60b7e0b4c65841e5c354fd8bc7
+    bin: kubectl-clusters
+    files:
+    - from: "kubectl-clusters*/kubectl-clusters"
+      to: .
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  caveats: |
+    This plugin needs the config file clusters.ini in current dir or refer with env CLUSTERS_CONFIG.
```

Comment:
You should also mention that you require a python runtime. You should also mention the minimum version required and potentially non-standard libraries that need to be installed.

Reply:
Python dependency caveat added